controllers/storagecluster/storageclasses.go

```diff
@@ -366,6 +370,10 @@ func newNonResilientCephBlockPoolStorageClassConfiguration(initData *ocsv1.Stora
 		},
 		isClusterExternal: initData.Spec.ExternalStorage.Enable,
 	}
+	if !(initData.GetAnnotations()[keyRotationEnableAnnotation] == "true") {
```

Comment:
```suggestion
initData.GetAnnotations()[keyRotationEnableAnnotation] == "false")
```
How about making it simple check and expect only false value?

Reply:
Done.